kit/score/results_test.go

```diff
@@ -118,8 +118,8 @@ func TestExtractResultsWithMultipleNonZeroScoreLines(t *testing.T) {
 		t.Fatalf("ExtractResult() expected 2 Score entries, got %d: %+v", len(res.Scores), res.Scores)
 	}
 	for _, score := range res.Scores {
-		if score.Score != -1 {
-			t.Errorf("ExtractResult() expected -1, got %d", score.Score)
+		if score.GetScore() != -1 {
```

Comment:
Seems like I cannot add comments to lines that are not changed with my VS Code extension, however:

Look for BuildInfo.BuildLog and BuildInfo.ExecTime in this file, both of these (BuildLog and ExecTime) have getters.